### ISSUE TITLE
Feature request: support the DD markup property

### ISSUE THREAD

**Issue by Trevoke** [SabakiHQ/Sabaki#906]
SGF has the ability to dim specific points/a region on the board ( https://www.red-bean.com/sgf/properties.html#DD ) and it would be very nice if that were supported by Sabaki!

As far as I can tell, this (and maybe SL, "selected points with unknown markup type") are the only two markup types from the SGF standard not yet supported by sabaki.

**Comment by HackYardo** [comment]
Is it possible to program a SGF markup system, so that users can add or delete their own markup symbols, i.e. any unicode char?

**Comment by Trevoke** [comment]
The "text" property let's people add arbitrary text labels of a few
characters, if that helps. In Sabaki it's "select the A then shift+click"
if I recall correctly.

Is this close to what you want?

--

Aldric.

**Comment by Trevoke** [comment]
Also your request is definitely not related to the first one, so if this
doesn't help you, I'd suggest opening a new issue so both concerns can be
handled separately.

--

Aldric.

**Comment by apetresc** [comment]
I don't believe we support the `VW` property related to the `DD` property either, but that one seems quite a bit harder to implement than `DD`. `DD` should be doable, thanks for the suggestion :)

**Comment by HackYardo** [comment]
Be careful, they are different.
- C[] is a Node Annotation Property
- VW[] is a Miscellaneous Property (for printing)
- DD[] is a Markup Property
- LB[] is a Markup Property for showing text(or unicode emoji) on board, thanks to @Trevoke !

**Comment by abaisero** [comment]
bumping this to add a similar request, since it was already mentioned, and it's pretty similar to DD:  it would be very nice if Sabaki had support for the SL property.  I'm working on a webapp for whole-board problems, and need a way to indicate subsets of vertices;  at first I was abusing other markers for this purpose, but that's a bit of a problem in its own right; I might have to switch to another editor just for this =(

**Comment by HackYardo** [comment]
@abaisero other editor? what is it? q5Go or Ogatak? Which editor supports all sgf properties?
